front/src/common/Tooltip.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from "react";
+import "./Tooltip.scss";
+
+type Props = {
+  msg?: string;
+};
+
+/**
+ * Icon displaying help text when hovered
+ * @param msg string
+ */
+const Tooltip = ({ msg }: Props) => {
+  const [display, setDisplay] = useState(false);
+  return !!msg ? (
+    <span className="tooltip__container">
+      <span
+        role="button"
+        className="tooltip__trigger"
+        onMouseOver={() => setDisplay(true)}
+        onMouseLeave={() => setDisplay(false)}
+      >
+        <img src="disposal-icon.jpg" height="32" alt="Aide" />
+      </span>
+
+      {display ? <p className="tooltip__content"> {msg} </p> : null}
```

Comment:
Tu peux utiliser la syntaxe suivante s'il n'y a pas d'else
```suggestion
      {display && <p className="tooltip__content"> {msg} </p>}
```

Reply:
done